src/app/static/src/app/store/downloadResults/downloadResults.ts

```diff
@@ -18,12 +18,6 @@ export const initialDownloadResults = {
     error: null
 }
 
-export enum DOWNLOAD_TYPE {
```

Comment:
Moved to avoid a was a circular dependency, now here: https://github.com/mrc-ide/hint/pull/663/files#diff-b4e97f9053896e4bab011155c7ed2ad40cf453d0da07ccbb80ae5144a9424c23R185